src/DeFuncto.Tests/Core/Types/Du7/Match.cs

```diff
@@ -71,52 +71,52 @@ int h
                 _ => $"{g}{a.Get}"
             ).Run(result => Assert.Equal($"{d.Get}{a.Get}", result));
 
-        [Fact(DisplayName = "Matches the Fourth option")]
-        public void OnFourth() =>
-            Fourth<int, int, int, string, int, int, int>("ban").Match(
-                val => $"{val}ana",
-                val => $"{val}ana",
-                val => $"{val}ana",
-                val => $"{val}ana",
-                val => $"{val}ana",
-                val => $"{val}ana",
-                val => $"{val}ana"
-            ).Run(result => Assert.Equal("banana", result));
+        [Property(DisplayName = "Matches the Fourth option")]
+        public void OnFourth(NonNull<string> a, NonNull<string> b) =>
```

Comment:
For all of these cases, add a third input for the ones that are not being matched.